apps/public-web/src/EShopOnAbp.PublicWeb/Pages/ProductDetail.cshtml

```diff
@@ -0,0 +1,55 @@
+﻿@page "{id:guid}"
```

Comment:
how about making url `/products/{id}` ?

Reply:
I follow the same approach with OrderReceived
```
@page "{orderNo:int}"
```